webpack.common.ts

```diff
@@ -37,18 +55,14 @@ module.exports = {
         'node_modules/monaco-languageclient/lib/vscode-compatibility'
       ),
     },
-    extensions: ['.tsx', '.ts', '.js', '.wasm'],
+    extensions: ['.tsx', '.ts', '.js', '.wasm', '...'],
```

Comment:
this `'...'`, addition was recommended by a webpack maintainer in a test repo as a best practice that automatically includes the default values for `extensions`